### PR TITLE
Bugfix/disable shake gesture

### PR DESCRIPTION
**Pull request by amobarak**
## Pull Request checklist:

- [x] [WIP] tag removed from PR title
- [x] PR has an understandable description

## Git feature branch checklist:

- [x] branch name comply with our branching strategy
- [x] git branch contains relevant JIRA ticket number
- [x] branch rebased against the latest develop
- [x] commits are squashed

## Sign off process checklist:

- [x] Code has been tested manually
- [x] Tests are passing
- [ ] PR link added to JIRA
- [ ] Reviewers selected in Github
- [ ] Any new reusable component/widget added to component library on Confluence

- [ ] Screenshot(s) captured on the physical device (if applicable)
- [ ] Tested by QA
- [ ] PO's approval

### CODE REVIEW

**Review by TomBillingtonUK — APPROVED** [review]

File: src/app/app.component.ts [hunk 1]
```diff
@@ -30,6 +31,10 @@ export class App {
         this.configureStatusBar();
         this.configureAccessibility();
         this.loadAppInfo();
+
+        if (cordova && cordova.plugins && cordova.plugins.DisableShakeToEdit) {
+          cordova.plugins.DisableShakeToEdit.pluginInitialize();
```
Comment (ammarhaiderbjss) [inline src/app/app.component.ts]: @amobarak Pretty sure we don't need this, the plugin gets initialised automatically

Comment (amobarak) [reply src/app/app.component.ts]: @Cavalier07 Any ideas?

Comment (TomBillingtonUK) [reply src/app/app.component.ts]: try it without?

**Review by TomBillingtonUK — APPROVED** [review]

File: src/karmaGlobalMocks.js [hunk 2]
```diff
@@ -22,6 +22,9 @@ const cordova = {
         cb(true);
       },
     },
+    DisableShakeToEdit: {
+      pluginInitialize: () => { },
```
Comment (ammarhaiderbjss) [inline src/karmaGlobalMocks.js]: Don't need this anymore either

**Review by ammarhaiderbjss — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**